tgui/packages/tgui/interfaces/IntegratedCircuit/index.js

```diff
@@ -98,14 +107,17 @@ export class IntegratedCircuit extends Component {
 
     this.handlePortDrag(event);
 
+    this.timeUntilPortReleaseWorks
+      = performance.now() + TIME_UNTIL_PORT_RELEASE_WORKS;
```

Comment:
You should use `Date.now()` instead, `performance` is a profiling api which might not be available in IE10 and below, and timers themselves have a nanosecond precision, but truncated to 100ns on IE due to Spectre.